web/src/components/auth/SelfRegSnackbar.tsx

```diff
@@ -5,8 +5,8 @@ import { Icon } from "@/components/njwds/Icon";
 import { AuthAlertContext } from "@/contexts/authAlertContext";
 import { AuthContext } from "@/contexts/authContext";
 import { IsAuthenticated } from "@/lib/auth/AuthContext";
+import { useConfig } from "@/lib/data-hooks/useConfig";
 import analytics from "@/lib/utils/analytics";
-import Config from "@businessnjgovnavigator/content/fieldConfig/config.json";
```

Comment:
Should this be a refactor to replace the file import with the hook?

Reply:
I'm imagining yes for good cleanness but its also not breaking or problematic in any way

Reply:
That's a great idea. I went ahead and created the ticket. We can discuss it during an upcoming gprooming session to see if it's needed.

https://www.pivotaltracker.com/story/show/185992659